package-lock.json

```diff
@@ -12024,7 +10644,7 @@
     },
     "node_modules/ethereumjs-abi": {
       "version": "0.6.8",
-      "resolved": "git+https://git@github.com/ethereumjs/ethereumjs-abi.git#ee3994657fa7a427238e6ba92a84d0b529bbcde0",
+      "resolved": "git+ssh://git@github.com/ethereumjs/ethereumjs-abi.git#ee3994657fa7a427238e6ba92a84d0b529bbcde0",
```

Comment:
Heads up here @nicosantangelo. IDKW every time you bump dcl-dapps this is changed and gitlab stop working. Can you just change `+ssh` to `+https` I always re-install or change it manually.